static/dtale/menu/ThemeOption.jsx

```diff
@@ -17,7 +18,7 @@ class ReactThemeOption extends React.Component {
     const { setTheme, theme } = this.props;
     const updateTheme = newTheme => () => serverStateManagement.updateTheme(newTheme, () => setTheme(newTheme));
     return (
-      <li className="hoverable" style={{ color: "#565b68" }}>
+      <MenuItem style={{ color: "#565b68" }} description={Descriptions.theme}>
```

Comment:
Since I'm looking at the theme option, have you tested your changes against "Dark Mode"?  I don't believe the theme really changes anything about the menu, but might want to validate nothing screwy happens.